packages/ag-charts-types/src/presets/sparkline/sparklineAxisOptions.ts

```diff
@@ -28,17 +28,17 @@ export interface AgSparklineContinuousAxisOptions<TDatum extends Date | number>
 }
 
 export interface AgSparklineCategoryAxisOptions extends AgSparklineBaseCategoryAxisOptions {
-    type: 'category';
+    type?: 'category';
```

Comment:
```suggestion
    type: 'category';
```

We should enforce it at the type level, but if they don't pass it in, we can just let it slide

Reply:
Updated with a different approach, let me know what you think

Reply:
Ok, my proposed doesn't fit the bill for other reasons - could you suggest a way to achieve the strong typing optionality you mention?

Reply:
Reverted